Solutions/Samsung Knox Asset Intelligence/SolutionMetadata.json

```diff
@@ -0,0 +1,16 @@
+{
+	"publisherId": "samsungelectronics1734042706970",
+	"offerId": "samsung-knox-asset-intelligence-sentinel",

```

Comment:
please include azure-sentinel in offerid

Reply:
The offerId field has been updated. Thanks!